java/WordCount.java

```diff
@@ -69,8 +71,13 @@ public static void main(String[] args) throws IOException {
         ArrayList<CountForWord> lst = new ArrayList<>(m.values());
         Collections.sort(lst);
         System.err.println("output...");
+        BufferedWriter outputWriter = new BufferedWriter(new OutputStreamWriter(System.out));
         for(CountForWord c : lst){
-            System.out.println(c.word + "\t" + c.count);
+            outputWriter.write(c.word);
```

Comment:
`outputWriter.write(c.word + "\t" + c.count);
 outputWriter.newLine();`

Otherwise it's going to interpret c.count as a char